lib/openai/assistants.rb

```diff
@@ -0,0 +1,23 @@
+module OpenAI
+  class Assistants
+    def initialize(client:)
+      @client = client.beta(assistants: "v1")
+    end
+
+    def list
+      @client.get(path: "/assistants")
+    end
+
+    def retrieve(id:)
+      @client.get(path: "/assistants/#{id}")
+    end
+
+    def create(parameters: {})
```

Comment:
Thanks for putting this together! Could we also add a `modify` method? 

```rb
def modify(id:, parameters: {})
  @client.json_post(path: "/assistants/#{id}", parameters: parameters)
end
```

Reply:
Yup, I've added that for messages, threads, assistants and runs.

Reply:
Thank you!